.sdk_metadata.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
It confuses me why this shows up in the diff, considering it is already present in this repo.